src/components/Article.jsx

```diff
@@ -1 +1,28 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, text, date }) => {
```

Comment:
maybe, it's better to move each property to the new line if there are 3 or more of them

Reply:
> maybe, it's better to move each property to the new line if there are 3 or more of them

I don't think that it is really necessary here